analysis/src/CompletionBackEnd.ml

```diff
@@ -1686,3 +1698,62 @@ Note: The `@react.component` decorator requires the react-jsx config to be set i
            Utils.startsWith name prefix
            && (forHover || not (List.mem name identsSeen)))
     |> List.map mkLabel
+  | CtypedContext (cp, typedContext) -> (
+    match typedContext with
+    | NamedArg argName -> (
+      (* TODO: Should probably share this with the branch handling CnamedArg... *)
+      let labels =
+        match
+          cp
+          |> getCompletionsForContextPath ~package ~opens ~rawOpens ~allFiles
+               ~pos ~env ~exact:true ~scope
+          |> completionsGetTypeEnv
+        with
+        | Some (typ, _env) ->
+          let rec getLabels ~env (t : Types.type_expr) =
```

Comment:
This must exist already. Doesn't matter for now.

Reply:
Yeah, it exists in another branch. I think it can be done in a better way, so can refactor that later.